.eslintrc.cjs

```diff
@@ -21,7 +21,7 @@ module.exports = {
 			rules: {
 				'import/no-named-as-default': 0,
 				'import/no-named-as-default-member': 0,
-			}
+			},
```

Comment:
😂 Are you committing with --no-verify??

Reply:
No time to verify ![](https://user-images.githubusercontent.com/1373867/216769654-1d933685-cb60-4e64-99c2-33e85186d35f.png)